src/mono/wasm/runtime/library_mono.js

```diff
@@ -601,9 +601,8 @@ var MonoSupportLib = {
 
 		mono_wasm_send_dbg_command_with_parms: function (id, command_set, command, command_parameters, length, valtype, newvalue)
 		{
-			const dataHeap = new Uint8Array (Module.HEAPU8.buffer, command_parameters, command_parameters.length);
-			dataHeap.set (new Uint8Array (this._base64_to_uint8 (command_parameters)));
-			this._c_fn_table.mono_wasm_send_dbg_command_with_parms_wrapper (id, command_set, command, dataHeap.byteOffset, length, valtype, newvalue.toString());
+			var dataHeap = this.mono_wasm_load_bytes_into_heap (this._base64_to_uint8 (command_parameters));
+			this._c_fn_table.mono_wasm_send_dbg_command_with_parms_wrapper (id, command_set, command, dataHeap, length, valtype, newvalue.toString());
```

Comment:
And `dataHeap` gets correctly passed here to the C function, same as `dataHeap.byteOffset`, I presume.

Reply:
Yes, as I run all the debugger-tests with this change :)
And this is used for all debugger commands.